### DISCUSSION TITLE
ERROR: raspbian-11: uWSGI not yet implemented

### DISCUSSION THREAD

**Discussion by un99known99** [searxng/searxng#2897] [Q&A] [answered]
Hi @return42
my previous #2845 was closed so I wanted to stress another question:

just wanted to give feedback, thx for the information with the upgrade script, it ran fine, updated to 2023.10.1 - I guess due to "rolling release" this should be last official commit?

I faced some "error" during upgrade about not installed / not implemented apache/nginx/uWSGI/pkg_install LXC_BASE_PACKAGES, this I would likely see as a "warn", the red "error" give a bad feeling :)

Despite that I guess that's it with upgrade task(s)?

**Comment by return42** [comment]
Not sure what error you mean, can you c&P the output here / thanks!

**Reply by un99known99** [reply]
@return42 

![grafik](https://github.com/searxng/searxng/assets/1615207/199c0286-d873-4827-ba2e-92f94a4192bd)

**Reply by return42** [reply, marked as answer]
That is because the script was not made for raspbian .. you can't install on raspbian with this script .. but may its good enough to update your instance.

The ["Step by Step"](https://docs.searxng.org/admin/installation-searxng.html#installation-basic) installation and the [Installation script](https://docs.searxng.org/admin/installation-scripts.html#installation-scripts) are only implemented "for" and tested "on" these distributions:

![image](https://github.com/searxng/searxng/assets/554536/4e62a57b-57fb-41c3-9749-236c3ca13095)

----

Related: https://github.com/searxng/searxng/pull/3515#issuecomment-2133245965
